src/NServiceBus.Metrics.ServiceControl.Tests/APIApprovals.Approve.approved.cs

```diff
@@ -3,5 +3,6 @@
     public class static MetricsOptionsExtensions
     {
         public static void SendMetricDataToServiceControl(this NServiceBus.MetricsOptions options, string serviceControlMetricsAddress, System.TimeSpan interval, string instanceId = null) { }
+        public static void SetServiceControlMetricsMessageTTBR(this NServiceBus.MetricsOptions options, System.TimeSpan timeToBeReceived) { }
```

Comment:
I know not everyone likes Fluent API but this is a really long method name. Are we sure we want this? If we add yet another method for some config, we might decide to have `SendMetricDataToServiceControl` return some `ServiceControlMetricsOptions` object after all and add all configuration on that one. Breaking the API, resulting in an upgrade to vNext, which isn't really possible, since we already have multiple versions for multiple versions of NServiceBus versions.

Not really sure what is the best solution here?

Reply:
Please, take a look at a comment made by @tmasternak https://github.com/Particular/NServiceBus.Metrics.ServiceControl/pull/14#discussion_r155448376

Reply:
Also, this is needed from the API parity point of view. That's the name of the method in 1.0.

Reply:
I'd prefer to have a `ServiceControlMetricsOptions` where we can put all of the SC-specific config. Is the v1 version already released? If so, then I say we leave it for now for the sake of consistency. We can always change it for the latest version and leave the older versions alone. It's a bit weird to release brand new components with an upgrade guide.

Reply:
The 1.0 PR was less readable because of the number of changes, comments, etc. That's why I didn't think about this earlier. But since this is already in 1.0, let's keep it like this.
I still wonder how we do breaking changes though. The NSB6.Heartbeats doesn't seem like such a bad idea from that perspective.